src/box/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ lua_source(lua_sources lua/schema.lua)
 lua_source(lua_sources lua/tuple.lua)
 lua_source(lua_sources lua/session.lua)
 lua_source(lua_sources lua/checkpoint_daemon.lua)
+lua_source(lua_sources lua/report_daemon.lua)
 lua_source(lua_sources lua/net_box.lua)
```

Comment:
Please find out the industry standard name for this module - talkback? And name the module accordingly. I think the word "daemon" is redundant, it doesn't convey too much extra info.